addons/sys_gui/CfgUIGrids.hpp

```diff
@@ -16,13 +26,19 @@ class CfgUIGrids {
                 };
             };
         };
-
         class Variables {
+            class GVAR(volumeControl) {
+                displayName = CSTRING(VolumeControlGrid_DisplayName);
+                description = CSTRING(VolumeControlGrid_Description);
+                preview = QPATHTOF(data\ui\volume_control_preview.paa);
+                saveToProfile[] = {0, 1};
+                canResize = 0;
```

Comment:
```suggestion
                canResize = 1;
```
Not sure if anything else has to be changed.

Reply:
Done. Can increase the default size if you think it should be bigger.

Reply:
I am with it being smaller, it's very visible with the rest of the changes. @IDI-Systems/acre-developers what do the rest of you think?